style.css

```diff
@@ -0,0 +1,19 @@
+ nav{
```

Comment:
It would be better if you would use one space of indentation between the class name and the opening tag. 
e.g nav {}